pegasus/test/test_section_api_helpers.rb

```diff
@@ -498,6 +507,35 @@ class SectionApiHelperTest < SequelTestCase
           assert_nil row[:script_id]
         end
       end
+
+      it 'replaces a course assignment with a script assignment' do
+        Dashboard.db.transaction(rollback: :always) do
+          course_id = FakeDashboard::COURSES[0][:id]
+          script_id = FakeDashboard::SCRIPTS[0][:id]
+
+          params = {
+            user: {id: 15, user_type: 'teacher'},
+            course_id: course_id
+          }
+          section_id = DashboardSection.create(params)
+          row = Dashboard.db[:sections].where(id: section_id).first
+          assert_equal course_id, row[:course_id]
```

Comment:
This was already asserted in "assigns a course to a section without one" and not part of this test. Aside: perhaps adding the `assert_nil row[:script_id]` to this test is appropriate?